### DISCUSSION TITLE
Can this be implemented with SPFX 1.4.1

### DISCUSSION THREAD

**Discussion by JohnRamminger** [n8design/htwoo#75] [General] [answered]
I have customers that are still on SharePoint 2019 and Subscription Service.

Can this be implemented with SPFX 1.4.1

This is an awesome project,  thanks for all you do.

John

**Comment by juliemturner** [comment, marked as answer]
With hTWOo Core - Yes 100%! Using core is effectively just html and css. You can look at Stefan's how-to guide on [getting started with SPFx](https://lab.n8d.studio/htwoo/how-to/how-to-spfx-react.html) and that should get you going. 

As far as hTWOo React and questioning if the project is backward compatible with ReactJS version 15.6.1, which I believe is the version that SPFx v 1.4 uses, I can't say for sure, but it's certainly possible. I would think that my helper classes might not work because of the version of Typescript I'm using in the library, but the components might. An issue you're going to see is that the install might give you trouble because the library's `package.json` indicates that you need version 16 or higher of ReactJS. You could install the project and then manually downgrade that value in the library's `package.json` file and then see if they work ok. Essentially if the components don't use something that wasn't available in ReactJS 15 then they should work, and I would certainly think it'd be worth a try.
